src/mumble/GlobalShortcut_macx.mm

```diff
@@ -147,8 +152,9 @@
 	                        GlobalShortcutMac::callback,
 	                        this);
 
-	if (! port) {
+	if (!port) {
 		qWarning("GlobalShortcutMac: Unable to create EventTap. Global Shortcuts will not be available.");
+		g.mw->msgBox(QLatin1String(osPrivacySettingsError));
```

Comment:
Does this work? I'm kinda doubtful ```g.mw``` exists at that point. From a quick check it looks like global shortcuts are created in the main window construction phase and before construction concludes g.mw is not set.

If that is the case maybe check ```QMessageBox``` for ways to display a modal without needing a parent. This would also allow you to get rid of the heavy includes.

Reply:
`g.mw->msgBox()` doesn't actually show a message box, it prints the message in the chat box.

Reply:
I see. Maybe we should rename it ;) Doesn't resolve the issue though.

Reply:
I agree, I initially though `msgBox()` was related to QMessageBox too.

Reply:
I tested the pull request.

Calling `g.mw->msgBox()` doesn't cause a crash, however the message is not shown because the chat box is not created yet:
```
QCoreApplication::postEvent: Unexpected null receiver
```